ColorApp/ViewControllers/ColorPickerViewController.m

```diff
@@ -211,26 +209,20 @@ - (IBAction)saveHue:(id)sender {
     UIAlertAction *saveAction = [UIAlertAction
                                actionWithTitle:NSLocalizedString(@"Save", @"Save action")
                                style:UIAlertActionStyleDefault
-                               handler:^(UIAlertAction *action)
-                               {
+                               handler:^(UIAlertAction *action) {
                                    UITextField *hueNameTextField = alertController.textFields.firstObject;
-                                   
-                                   NSLog(@"Save action");
+                                   NSString *hueName;
                                    if ([hueNameTextField.text isEqualToString:@""]) {
```

Comment:
After research I found out `isEmpty` isn't implemented by default and will have to be manually implemented. Putting a story on the backlog to generate and test an `isEmpty` method for `NSString`